lib/router/router.js

```diff
@@ -224,6 +224,8 @@ function SingleAppRouter(appdir, runas, prefix, logDir, settings) {
     var appSpec = new AppSpec(self.$appdir, self.$runas, self.$prefix, 
       self.$logDir, settings);
 
+    appSpec.logAsUser = settings.logAsUser;
```

Comment:
You're passing `settings` down the the `AppSpec` constructor -- why not just set `logAsUser` within?